build-system/tasks/helpers.js

```diff
@@ -393,7 +393,10 @@ async function finishBundle(
     );
     endBuildStep(logPrefix, `${destFilename} → ${latestName}`, startTime);
   } else {
-    endBuildStep(logPrefix, destFilename, startTime);
+    const loggingName = destFilename.startsWith('component-')
```

Comment:
We'll eventually be building `custom-element`, too. Seems like we could just check if `!destFileName.startsWith('amp')`?

Reply:
Can do, but we don't want to include non-extension targets, so I've also added a check for the presence of `options.npm`.